src/components/ScriptExplorer/OutputsForm.jsx

```diff
@@ -303,6 +303,9 @@ class OutputsForm extends React.Component {
                 <TextField
                   fullWidth
                   value={feeRate}
+                  type="number"
+                  minimum={1}
+                  step={1}
```

Comment:
I'm not sure how I feel about the step and the minimum.  On the one hand it seems to help prevent the issue but on the other hand a fee of 0.1 sats/byte is completely valid as is 0 sats/byte.  

I do think that setting the fee to < 1 sat/byte is a rare use case, however, and perhaps is something we could look into supporting only if we get users requesting it.

@destrys do you have any thoughts on this?  I know we were discussing this at one point in the past.

Reply:
Note that validation is also done before you can move on to signing. So even though the number can still be below 1 and greater than 0 if done manually, you’ll get an error that the fee is too low (it will warn if the amount is lower than it could be with 1 sat per byte for an empty tx). I agree it’s still not perfect, but the next step to avoid this would be adding more robust form validation to prevent such entry in the first place, which felt out of scope to me.